src/stories/Quiz/Hooks/index.tsx

```diff
@@ -164,10 +162,10 @@ export default function HooksTemplate(args) {
                 <p>{currentQuestionData.description}</p>
               </div>
               <div className='cio-question-options-container'>
-                {currentQuestion.next_question?.options?.map((option: QuestionOption) => (
+                {currentQuestionData.options.map((option: QuestionOption) => (
```

Comment:
double checking, @esezen : will `currentQuestionData.options` always be an array whenever `currentQuestionData` is truthy?